src/bootstrap/config.rs

```diff
@@ -538,14 +538,14 @@ impl Config {
         if let Some(ref rust) = toml.rust {
             debug = rust.debug;
             debug_assertions = rust.debug_assertions;
-            debuginfo = rust.debuginfo;
-            debuginfo_lines = rust.debuginfo_lines;
-            debuginfo_only_std = rust.debuginfo_only_std;
-            debuginfo_tools = rust.debuginfo_tools;
+            debuginfo_level = rust.debuginfo_level;
+            debuginfo_level_rustc = rust.debuginfo_level_rustc;
+            debuginfo_level_std = rust.debuginfo_level_std;
+            debuginfo_level_tools = rust.debuginfo_level_tools;
+            debuginfo_level_tests = rust.debuginfo_level_tests;
```

Comment:
Probably need to verify that the options are in the range `0-2` to fail early.

Reply:
Up to you. I personally think it's fine to wait; we'll fail pretty quickly anyway...

Reply:
Yeah, fails quickly and the message from the compiler is good, marking as resolved.